README.md

```diff
@@ -112,31 +112,39 @@ invoke changelog
 Once installed the application is bound to `pykiso`, it can be called with the following arguments:
 
 ```bash
-Usage: pykiso [OPTIONS] [PATTERN]
+Usage: pykiso [OPTIONS]
 
-  TAG Filters: Just add pairs of arguments to the pykiso call. Multiple values
-  for a key must be seperate via a comma.
+  Embedded Integration Test Framework - CLI Entry Point.
 
-  For example: pykiso -c your_config.yaml --branch dev,master --variant delta
+  TAG Filters: any additional option to be passed to the test as tag through
+  the pykiso call. Multiple values must be separated with a comma.
+
+  For example: pykiso -c your_config.yaml --branch-level dev,master --variant
+  delta
 
 Options:
   -c, --test-configuration-file FILE
                                   path to the test configuration file (in YAML
                                   format)  [required]
-
   -l, --log-path PATH             path to log-file or folder. If not set will
                                   log to STDOUT
-
   --log-level [DEBUG|INFO|WARNING|ERROR]
                                   set the verbosity of the logging
-  --version                       Show the version and exit.
-  -h, --help                          Show this message and exit.
   --junit                         enables the generation of a junit report
   --text                          default, test results are only displayed in
                                   the console
+  --step-report PATH              generate the step report at the specified
+                                  path
   --failfast                      stop the test run on the first error or
                                   failure
-  -v, --verbose                   enables the internal logs of the framework
+  -v, --verbose                   activate the internal framework logs
+  -p, --pattern TEXT              test filter pattern, e.g. 'test_suite_1.py'
+                                  or 'test_*.py'. It will be applied to all
+                                  defined test suites.
+  -t, --tc_filter TEXT            only run tests with names that match the
```

Comment:
I don't find -t very explicit to be honest, we could use `-f` instead for filter or do it in the pytest-style with `-k`

Reply:
ye we have different filters i gave it an explicit name --tc-filter   like test case filter
Pattern  could acutally be --tf-filter    test file filter

I thought also about -k but i didn't found a memorable name which starts with k to filter test cases